16.2.11 Release notes (#487)
ceph/ceph.io: src/en/news/blog/2023/v16-2-11-pacific-released/index.md

---
title: "v16.2.11 Pacific released"
date: "2023-01-26"
author: "yuriw"
tags:
  - "release"
  - "pacific"
---

This is the eighth backport release in the Pacific series. We recommend all users update to this release.

## Notable Changes

- Cephfs: The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard
  'AT_STATX_DONT_SYNC' macro. The 'AT_NO_ATTR_SYNC' macro will be removed in
  the future.

- Trimming of PGLog dups is now controlled by the size instead of the version.
  This fixes the PGLog inflation issue that was happening when the on-line
  (in OSD) trimming got jammed after a PG split operation. Also, a new off-line
  mechanism has been added: `ceph-objectstore-tool` got `trim-pg-log-dups` op
  that targets situations where OSD is unable to boot due to those inflated dups.
  If that is the case, in OSD logs the "You can be hit by THE DUPS BUG" warning
  will be visible.
  Relevant tracker: https://tracker.ceph.com/issues/53729

> - RBD: `rbd device unmap` command gained `--namespace` option.  Support for
>   namespaces was added to RBD in Nautilus 14.2.0 and it has been possible to
>   map and unmap images in namespaces using the `image-spec` syntax since then
>   but the corresponding option available in most other commands was missing.

Reviewer: ```suggestion
- RBD: The `rbd device unmap` command gained the `--namespace` option.  Support for
  namespaces was added to RBD in Nautilus 14.2.0; while it had been possible to
  map and unmap images in namespaces using the `image-spec` syntax since then,
  the corresponding option available in most other commands was missing.
```

## Changelog

- <span></span>.github/CODEOWNERS: tag core devs on core PRs ([pr#46520](https://github.com/ceph/ceph/pull/46520), Neha Ojha)

- <span></span>.github: continue on error and reorder milestone step ([pr#46448](https://github.com/ceph/ceph/pull/46448), Ernesto Puerta)

- <span></span>.readthedocs<span></span>.yml: Always build latest doc/releases pages ([pr#47443](https://github.com/ceph/ceph/pull/47443), David Galloway)

> - [bluestore] Improve deferred write decision ([pr#49170](https://github.com/ceph/ceph/pull/49170), Adam Kupczyk, Igor Fedotov)
> 
> - [pacific] mgr/alerts: Add Message-Id and Date header to sent emails ([pr#46312](https://github.com/ceph/ceph/pull/46312), Lorenz Bausch)

Reviewer: ```suggestion
- bluestore: Improve deferred write decision ([pr#49170](https://github.com/ceph/ceph/pull/49170), Adam Kupczyk, Igor Fedotov)

- mgr/alerts: Add Message-Id and Date header to sent emails ([pr#46312](https://github.com/ceph/ceph/pull/46312), Lorenz Bausch)
```

And then alphabetized
